scripts/fetch-schemas.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
 
 REPO_ROOT=$(git rev-parse --show-toplevel)
-BRANCH_OR_TAG="${1:-v1.0}"
```

Comment:
In this shell script, the `BRANCH_OR_TAG` value is used to construct a url with this form:
```
https://raw.githubusercontent.com/interledger/open-payments/$BRANCH_OR_TAG/openapi/(schemas/auth-server/resource-server).yaml
```

Unfortunately, `https://raw.githubusercontent.com/interledger/open-payments/v5.0.0/openapi/(schemas/auth-server/resource-server).yaml` doesn't resolve to a valid URL. It needs to be point to [this release](https://github.com/interledger/open-payments/releases/tag/%40interledger%2Fopen-payments%405.0.0).

Reply:
Gotcha, I misunderstood. I've gone ahead and updated the link and verified that it works with the `pnpm fetch-schemas` command.

Reply:
You're right. It was like that for version 1.0, but I removed it when I updated the link. I've put the `1:-` back in now so that tags can be passed again.